aiven/client/cli.py

```diff
@@ -5000,6 +5000,89 @@ def application_user__delete(self) -> None:
         self.client.delete_application_user(organization_id=self.args.organization_id, user_id=self.args.user_id)
         print(f"Application user {self.args.user_id} deleted successfully.")
 
+    @arg.json
+    @arg.force
+    @arg.organization_id
+    @arg("--resource-type", choices=["organization", "project"], help="Resource type", required=True)
+    @arg("--project-id", help="Project ID, required if resource_type is 'project'")
+    @arg(
+        "--permissions",
+        action="append",
+        default=[],
+        help="Permissions to assign to the application user. Can be specified multiple times to add multiple permissions",
+    )
+    @arg("user_id", help="Application User ID")
+    def application_user__permissions__set(self) -> None:
```

Comment:
The API invoked is not limited to application users (which from this perspective are just like normal users), it making the command just `permissions__set` and allowing `--principal-type` and `--principal-id` to allow setting these for users and groups.